app/services/feeds/assemble_article_markdown.rb

```diff
@@ -31,6 +31,10 @@ def call
 
     private
 
+    def processed_title
+      @title.size <= 128 ? @title : "#{@title[0..124]}..."
```

Comment:
Rails has this built-in:

```ruby
test = "x" * 20
test.truncate(10, omission: "...")
#=> "xxxxxxx..."
```

```suggestion
      @title.truncate(128, omission: "...")
```

Reply:
Well, having named it, this is obviously the right thing. :smile:
I'll do the same for `Article#processed_description` in a second PR.

The only thing we may be doing "specially" in processed description is aligning the ellipsis to non-whitespace (so you don't get "text text text ..." and instead get "text text text..." even if it causes the truncated string to be shorter than it would be). It looks like `separator: " "` is what I also want for that behavior.